ios/Sources/TimetableFeature/TimetableDetailScreen.swift

```diff
@@ -0,0 +1,245 @@
+import Component
+import ComposableArchitecture
+import Model
+import Styleguide
+import SwiftUI
+
+public struct TimetableDetailScreen: View {
+    enum Const {
+        static let imageSize: CGFloat = 40
+        static let iconSize: CGFloat = 20
+        static let footerIconSize: CGFloat = 24
+    }
+
+    let store: Store<ViewState, ViewAction>
+
+    struct ViewState: Equatable {
+        var timetable: TimetableItem
+    }
+
+    enum ViewAction {}
+
+    public var body: some View {
+        ZStack {
+            ScrollView {
+                WithViewStore(store) { viewStore in
+                    Text(viewStore.timetable.title.jaTitle)
+                        .font(.title)
+                        .foregroundColor(AssetColor.Base.primary.color)
+                        .lineLimit(nil)
+                        .padding(.bottom, 12)
+
+                    speakers(viewStore.timetable.speakers)
+
+                    descriptions(timetable: viewStore.timetable)
+
+                    body(timetable: viewStore.timetable)
+                }
+                .padding(.horizontal, 16)
+            }
+
+            fixedFooter(
+                tapSlide: {
+                    // TODO: FIXME
+                },
+                tapVideo: {
+                    // TODO: FIXME
+                }
+            )
+        }
+    }
+}
+
+private extension TimetableDetailScreen {
+    func speakers(_ speakers: [Speaker]) -> some View {
+        VStack(alignment: .leading, spacing: 12) {
+            ForEach(speakers, id: \.self) { speaker in
+                HStack {
+                    ImageView(
+                        imageURL: URL(string: speaker.iconURLString),
+                        placeholderSize: .medium,
+                        width: Const.imageSize,
+                        height: Const.imageSize
+                    )
+                    .clipShape(Circle())
+                    .overlay(
+                        RoundedRectangle(cornerRadius: Const.imageSize / 2)
+                            .stroke(AssetColor.Separate.image.color, lineWidth: 1)
+                    )
+                    Text(speaker.name)
+                        .font(.body)
+                        .foregroundColor(AssetColor.Base.secondary.color)
+
+                    Spacer()
+                }
+            }
+        }
+        .padding(.bottom, 24)
+    }
+
+    func descriptions(timetable: TimetableItem) -> some View {
+        VStack(alignment: .leading, spacing: 6) {
+            ForEach(DesciptionType.allCases, id: \.self) { type in
+                HStack(spacing: 11) {
```

Comment:
Maybe `8` is correct value 🙏🏼 
```suggestion
                HStack(spacing: 8) {
```
<img width="284" alt="スクリーンショット 2021-09-21 3 25 05" src="https://user-images.githubusercontent.com/30540303/134054491-0fca7358-dbcf-4ea0-a6fc-c43606499880.png">